keyboards/keebio/viterbi_numpad/readme.md

```diff
@@ -0,0 +1,23 @@
+Viterbi (numpad variant)
+========================
+
+A repurposing of the right half of a regular Viterbi (rotated) for use as a numpad with a few extra keys.
+This is probably not useful for anybody apart from me, but I have a spare right PCB and need a numpad.
+
+I haven't included a rev1 version, since I only have a rev2.
+
+Keyboard Maintainer: [Josh Channings](https://github.com/joshch)
+Hardware Supported: Elite C
+Hardware Availability: [Keebio](https://keeb.io)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```